hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/fees/FeesAndRatesProvider.java

```diff
@@ -104,7 +104,8 @@ public ExchangeRate rates() {
 
     public void updateRateSet(ExchangeRateSet newSet) {
         rateSet = newSet;
-        String message = String.format("Updating rates! Now :: %s", rateSetAsString(newSet));
+        String newSetAsString = rateSetAsString(newSet);
```

Comment:
what is the reason for this change?

Reply:
it was causing the E2E test failing and the original change was becuase of the main topic that I fix all this changes
[Smoke code: fixed formatting issue over strings in logs and concatenation Printf-style format strings should be used correctly java:S3457](https://github.com/hashgraph/hedera-services/pull/5704#top)

Reply:
I don't understand how this would affect whether a test passes or not. What was failing that this fixes?
Also these temporary variables should be declared final.

Reply:
I changed it to final.
      
this is how it looked in `develop` branch
```
        rateSet = newSet;
        log.info("Updating rates! Now :: " + rateSetAsString(newSet));
```
I try to apply format on the content of it and it started to fail 6 e2e test. I am ready to try it agin till I break it up for 2 strings to remove the code smell.